File: src/main/kotlin/org/kite9/diagram/dom/transform/LeafTransformer.java
package org.kite9.diagram.dom.transform;

import org.kite9.diagram.dom.painter.LeafPainter;
import org.kite9.diagram.model.position.Dimension2D;

public interface LeafTransformer extends SVGTransformer {

	public Dimension2D getBounds(LeafPainter p);
}

File: kite9-visualization/src/commonMain/kotlin/org/kite9/diagram/dom/transform/LeafTransformer.kt
package org.kite9.diagram.dom.transform

import org.kite9.diagram.dom.painter.LeafPainter
import org.kite9.diagram.model.position.Dimension2D

interface LeafTransformer : SVGTransformer {

    fun getBounds(p: LeafPainter): Dimension2D
}